src/purejavahidapi/windows/CfgmgrLibrary.java

```diff
@@ -32,10 +32,10 @@
 import com.sun.jna.Library;
 import com.sun.jna.Native;
 import com.sun.jna.win32.W32APIOptions;
-
+/*

```

Comment:
Should this file be removed?

Reply:
Yes - just waited for approval. Done now.